OMEdit/OMEditGUI/Animation/AbstractAnimationWindow.cpp

```diff
@@ -129,6 +130,11 @@ void AbstractAnimationWindow::openAnimationFile(QString fileName)
         mpPerspectiveDropDownBox->setCurrentIndex(1);
         cameraPositionSide();
       }
+
```

Comment:
We only need stashing for composite models not for Modelica models. Perhaps we should send some boolean to `openAnimationFile` which tells whether to stash or not.